File: tests/dynamic_fixtures/php/xss_negative.php
<?php
// XSS — negative fixture.
// Safe: uses htmlspecialchars() before output.
// Entry: renderPage($userInput)  Cap: HTML_ESCAPE
// Expected verdict: NotConfirmed

function renderPage($userInput) {
    $safe = htmlspecialchars($userInput, ENT_QUOTES, 'UTF-8');
    echo '<html><body>' . $safe . '</body></html>' . "\n";
}

File: tests/dynamic_fixtures/go/xss_negative.go
// XSS — negative fixture.
// Safe: uses html.EscapeString before output.
// Entry: RenderPage(userInput string)  Cap: HTML_ESCAPE
// Expected verdict: NotConfirmed

package entry

import (
	"fmt"
	"html"
)

func RenderPage(userInput string) {
	safe := html.EscapeString(userInput)
	fmt.Print("<html><body>" + safe + "</body></html>\n")
}
